app/views/search.nunjucks

```diff
@@ -15,6 +15,9 @@
         <!-- label -->
       </label>
 
+      {% if errorMessage %}
```

Comment:
AFIAK this is the way the browser works and until such a time that JS functionality is added to fix the issue on the front end there isn't much to be done about it. And I'm not inclined to do that as a priority now.
Happy to hear about alternatives.